src/ansys/dpf/core/operators/mapping/find_reduced_coordinates.py

```diff
@@ -110,11 +111,12 @@ def _spec():
                 200: PinSpecification(
                     name="use_quadratic_elements",
                     type_names=["bool"],
-                    optional=False,
+                    optional=True,
                     document="""If this pin is set to true, reduced
         coordinates are computed on the
         quadratic element if the element is
-        quadratic (default is false).""",
+        quadratic (more precise but less
+        performant). default is false.""",
```

Comment:
Hey @rafacanton, a small typo here, should be `Default is false.`
Do you know who made this change?

Reply:
@PProfizi I did. It is in capital letters in the operator [spec](https://tfs.ansys.com:8443/tfs/ANSYS_Development/DPF/_git/DPF?path=/Ans.Dpf.FEMUtils/EvaluationOperators.cpp&version=GBdevelop&line=29&lineEnd=29&lineStartColumn=42&lineEndColumn=59&lineStyle=plain&_a=contents), so this is an issue in the operators generation